palindrom-client.html

```diff
@@ -427,7 +399,7 @@
                                     templateDomBind.notifySplices(parentsPolymerPath,
                                         [{
                                             index: parseInt(name, 10),
-                                            removed: [result],
+                                            removed: [result && result.removed],
```

Comment:
I guess it should be `(result && [result.removed]) || []`, because [docs demand an empty array](https://www.polymer-project.org/1.0/docs/api/classes/Polymer.Base) (not `undefined`) when nothing is removed.